recipes/libuv/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "libuv" %}
+{% set version = "1.9.1" %}
+{% set sha256 = "a6ca9f0648973d1463f46b495ce546ddcbe7cce2f04b32e802a15539e46c57ad" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/{{ name }}/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```

Comment:
Though for Windows we really do need Python for all of them. So it should be something like `skip: true  # [not win and not py27]`.